packages/common/src/constraints/ValidityChecks.ts

```diff
@@ -438,3 +439,45 @@ export const checkBeefyKeys = async (
     throw new Error("could not make validity check");
   }
 };
+
+// Checks if the candidate is in a sanctioned location
+export const checkSanctionedGeoArea = async (
+  config: Config.ConfigSchema,
+  candidate: Candidate,
+): Promise<boolean> => {
+  try {
+    const location = await queries.getCandidateLocation(
+      candidate.slotId,
+      candidate.name,
+    );
+    if (location && location.region && location.country) {
+      const sanctionedCountries = config.constraints?.sanctionedCountries;
+      const sanctionedRegions = config.constraints?.sanctionedRegions;
+      if (
+        sanctionedCountries.includes(location.country) ||
```

Comment:
just one last minor thing here: 
given these two lists
["RU", "IR", "CU", "KP", "SY"]
["Crimea", "Luhansk", "Donetsk"]
don't you think is better to trim and lower capitalize the check (both the sanctionedCountry/sanctionedRegion and the string from the config) ? Otherwise if the source will give you back "CRIMEA" or "crimea" you won't have an hit
wdyt ? 

(Don't change the 2 lists, eventually just add a trim() and a toLowerCase() function)

Reply:
Good call, will do.